src/plugins/intel_cpu/src/edge.cpp

```diff
@@ -272,11 +272,30 @@ void Edge::allocate(const void* mem_ptr) {
 
     auto parentPtr = getParent();
     memoryPtr.reset(new Memory(parentPtr->getEngine()));
-
-    memoryPtr->Create(inputDesc, mem_ptr, false);  // no pads zeroing
+    allocate(memoryPtr, inputDesc);
     status = Status::Allocated;
 }
 
+void Edge::allocate(const void* mem_ptr) {
+    auto allocateFunc = [&](const MemoryPtr& memoryPtr, const MemoryDesc& inputDesc) {
+        memoryPtr->Create(inputDesc, mem_ptr, false);  // no pads zeroing
```

Comment:
Reference in lambda to local variable `mem_ptr` is risky. It doesn't fail in this case as `allocateCommon` calls the lambda before it's destroyed. Consider 'by value'.

Reply:
yes, you are right.